extractor/filesystem/language/python/internal/pypipurl/pythonpurl.go

```diff
@@ -0,0 +1,30 @@
+// Package pypipurl converts an inventory to a PyPI type PackageURL.
+// TODO(#173): Replace with purl.New() which will contain mapping of all ecosystems
```

Comment:
Hm, with internal bugs one can just paste the link to directly get to the bug, for github issues that doesn't work as well.

How does OSV currently refer to issues in TODOs? Is there a way to embed a short link that goes to the issue page directly?

Reply:
As far as I know, using a # then the issue number is generally how you get uniquely identified issues on github. I've also added the b/ link as well.